ports/raspberrypi/common-hal/microcontroller/Pin.c

```diff
@@ -28,18 +28,34 @@
 
 #include "common-hal/microcontroller/__init__.h"
 #include "shared-bindings/microcontroller/Pin.h"
+#include "bindings/cyw43/__init__.h"
```

Comment:
Guard with `CIRCUITPY_CYW43` so it's completely not there if compiling for regular Pi Pico?

Reply:
done